tests/testthat/test-driver-sql-server.R

```diff
@@ -58,6 +58,7 @@ test_that("SQLServer", {
     "list_fields_wrong_table",
     "list_fields_quoted",
     "list_fields_object",
+    "list_objects_features",
```

Comment:
I'm not sure why this worked before, but it was skipped on all other backends.

Reply:
Yep - looks like fixed by @krlmlr on `DBItest::main`.  Maybe we bring back once that one makes its way to cran